ui/src/app/components/runtime-configuration-panel.spec.tsx

```diff
@@ -907,40 +928,43 @@ describe('RuntimeConfigurationPanel', () => {
     expect(wrapper.find(WarningMessage).exists()).toBeFalsy();
   });
 
-  it('should warn user about reboot if there are updates that require one - CPU', async () => {
```

Comment:
the new 2 tests are good, but we have lost coverage for the conditions described by these 2 tests.  Please add 2 new ones to restore coverage.

Reply:
done